hive-metastore/src/test/java/org/apache/iceberg/hive/TestHiveMetastore.java

```diff
@@ -242,6 +242,7 @@ private void initConf(HiveConf conf, int port) {
     conf.set(HiveConf.ConfVars.METASTORE_TRY_DIRECT_SQL.varname, "false");
     conf.set(HiveConf.ConfVars.METASTORE_DISALLOW_INCOMPATIBLE_COL_TYPE_CHANGES.varname, "false");
     conf.set("iceberg.hive.client-pool-size", "2");
+    conf.set(HiveConf.ConfVars.HIVE_TXN_TIMEOUT.varname, "1s");
```

Comment:
Is it intentional to change this for all of the tests?
I would keep this as a default here, and modify the config only in the `TestHiveComitLocks` test